Examples/ByFramework/MSTest/MSTest.FSharp/Stock.feature

```diff
@@ -0,0 +1,13 @@
+﻿Feature: Refunded or replaced items should be returned to stock
+
+Scenario: Refunded items should be returned to stock
+	Given a customer buys a black jumper
+	And I have 3 black jumpers left in stock 
+	When he returns the jumper for a refund 
+	Then I should have 4 black jumpers in stock
+
+Scenario: Refunded items should be returned to stock
```

Comment:
You have two same scenarios. That is probably unwanted (and it differs from other test frameworks).

Reply:
From the same reason it is actually failing to execute the tests using .NET Framework - the emitted type is named according to the scenario name. If you have two scenarios with the same name, then it fails to emit two types with the same name. Is this a thing we want to solve (like randomizing the type name)?

Reply:
@mchaloupka I fixed the scenario but I have also found that you actually broke it with the parsing changes. The code worked before and the reason is that the code named second instance of such scenario "Scenario: Refunded items should be returned to stock~2". I think we should make it backward compatible. Will you look on it, please?